models/fivetran_log__connector_status.sql

```diff
@@ -56,9 +54,9 @@ connector_metrics as (
         max(case when event_type = 'WARNING' then connector_log.created_at else null end) as last_warning_at
 
     from connector_log 
-        right join connector on connector_log.connector_name = connector.connector_name
-        and connector_log.destination_database = connector.destination_database
-    group by 1,2,3,4,5,6,7
+        right join connector 
```

Comment:
why right join instead of left?

```
from connector
left join connector_log
```

Reply:
i think truly no particular reason.... should i change to left for consistency?